facebookads/api.py

```diff
@@ -126,10 +129,12 @@ def _evaluate_if_transient(self):
             is_transient = error.get('is_transient', False)
             error_message = error.get('message', False)
 
-            if self.is_failure() and not is_transient and error_message == self.TRANSIENT_ERROR_MESSAGE:
-                error['is_transient'] = True
-                json_body['error'] = error
-                self._body = json.dumps(json_body)
+            for transient_error_message in self.TRANSIENT_ERROR_MESSAGES:
+                if self.is_failure() and not is_transient and transient_error_message in error_message:
+                    error['is_transient'] = True
+                    json_body['error'] = error
+                    self._body = json.dumps(json_body)
+                    break
```

Comment:
Uhm, what?!

```
if (self.is_failure() and not is_transient and any((transient_message in error_message
        for transient_message in self.TRANSIENT_ERROR_MESSAGES))):
    error['is_transient'] = True
    json_body['error'] = error
    self._body = json.dumps(json_body)
```

(check what pip recommends for indentation of the various parts of the `if` statement)